default/scripting/buildings/colonies/col_bld_gen.py

```diff
@@ -211,19 +212,22 @@
     sp_id = species[0]
     sp_name = sp_id.split("_", 1)[1]
     sp_desc_name = species[1]
+    sp_tags = '"'+sp_id+'"'
     sp_graphic = species[2]
     sp_filename = sp_id + ".focs.txt"
     with open(os.path.join(outpath, sp_filename), "w") as f:
         if sp_id == "SP_EXOBOT":
-            f.write(t_main.substitute(id=sp_id, name=sp_name, graphic=sp_graphic, cost=70, time=5,
+            sp_tags += ' "CTRL_ALWAYS_REPORT"'
+            f.write(t_main.substitute(id=sp_id, name=sp_name, tags=sp_tags, graphic=sp_graphic, cost=70, time=5,
```

Comment:
Same goes for lines you change. When there is a style violation in the line you change try to fix it but no more.  So in this case please add spaces around the `=` operator.